test/e2e/util/velero/velero_utils.go

```diff
@@ -179,6 +179,7 @@ func checkRestorePhase(ctx context.Context, veleroCLI string, veleroNamespace st
 		return err
 	}
 	if restore.Status.Phase != expectedPhase {
+		fmt.Println(restore.Status.FailureReason)
```

Comment:
Please put more details information into this log output.

Reply:
removed, was just put for some debugging.